project/layersAfterAddingNewLayer.json

```diff
@@ -0,0 +1,162 @@
+[
+   {
+      "@id":1743099227843,
+      "@type":"Layer",
```

Comment:
@id and @type happening in import will go away eventually, but we will still need an id here.